src/test/cli/rbd/invalid-snap-usage.t

```diff
@@ -89,6 +89,7 @@
   rbd: snapname specified for a command that doesn't use it
   [22]
   $ rbd bench-write foo@snap
+  rbd: bench-write is deprecated, use rbd bench --io-type write ...
```

Comment:
No, it is not about "bench-write is deprecated" but about "snapname specified for a command that doesn't use it" error this test actually written for. This error is valid only for write type, but right now it rises for both `--io-type write` and `--io-type read`. For the second one I think we should allow to run bench.

I.e. you should pass `SNAPSHOT_PRESENCE_NONE` flag to `get_pool_image_snapshot_names()` only for write io type.